scrapers/BangBros.yml

```diff
@@ -15,7 +15,7 @@ xPathScrapers:
       Performers:
         Name: //div[@class="vdoCast"]/a[position()>1]/text()
       Image:
-        selector: //video/@poster
+        selector: //img[@id="player-overlay-image"]/@src
```

Comment:
on several urls the specific src gives "file not found"
thats why they failover using js to the image  url that selector video/@poster returns

urls that get no image
`https://bangbros.com/video64036/amia-miley-and-junior`
`https://bangbros.com/video25480/photo-shoot-fuck-fest`
`https://bangbros.com/video50959/fishing-for-cock`
`https://bangbros.com/video55225/pochatonas-and-the-skin-flute`
mostly older scenes

Reply:
You are right, i tested my selector for like 50 scenes never had problem, but it look like some image no longer exist from bangbros mostly on older scenes.
Added my selector as comment so people can choice.